daemon/container.go

```diff
@@ -34,13 +34,13 @@ import (
 )
 
 var (
-	ErrNotATTY                 = errors.New("The PTY is not a file")
-	ErrNoTTY                   = errors.New("No PTY found")
-	ErrContainerStart          = errors.New("The container failed to start. Unknown error")
-	ErrContainerStartTimeout   = errors.New("The container failed to start due to timed out.")
+	// ErrContainerRootfsReadonly is returned when a container
+	// rootfs is marked readonly. (perhaps move to archive.go and
+	// make private, the only site of usage)
```

Comment:
I think it's good practice to export the errors that get returned so package users can check for them, so I'd lean towards leaving it exported. Moving it to the file where it's used sounds okay but I don't think it matters much.

ErrRootFSReadOnly sounds good.